src/applications/ask-va/containers/DashboardCards.jsx

```diff
@@ -1,23 +1,35 @@
 import {
   VaAlert,
+  VaCard,
+  VaLink,
+  VaLoadingIndicator,
```

Comment:
Unless you're needing to use a callback function on an event or something else react-y, you shouldn't need to use/import the React binding of a web component. Just using `va-card`, `va-link`, or `va-loading-indicator` in the markup. 

There is a small performance gain by using the vanilla web component if possible.

Reply:
Noted for the follow up ticket for Dashboard